opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -199,67 +223,62 @@ message HistogramValue {
     // Exemplars are example points that may be used to annotate aggregated
     // Histogram values. They are metadata that gives information about a
     // particular value added to a Histogram bucket.
-    message Exemplar {
-      // Value of the exemplar point. It determines which bucket the exemplar
-      // belongs to.
-      double value = 1;
 
-      // The observation (sampling) time of the above value.
-      google.protobuf.Timestamp timestamp = 2;
+    // Value of the exemplar point. It determines which bucket the exemplar belongs to.
+    double exemplar_value = 2;
 
-      // Contextual information about the example value.
-      map<string, string> attachments = 3;
-    }
+    // The observation (sampling) time of the above value.
+    int64 exemplar_timestamp_unixnano = 3;
 
-    // Exemplars are example points that may be used to annotate aggregated
-    // Histogram values.
-    Exemplar exemplar = 2;
+    // timestamp_unspecified must be set to true if timestamp_unixnano value is unspecified.
+    // In that case timestamp_unixnano value will be decided by the backend.
+    bool exemplar_timestamp_unspecified = 4;
+
+    // exemplar_attachments are contextual information about the example value.
+    repeated StringKeyValuePair exemplar_attachments = 5;
```

Comment:
Add a comment that these need unique keys?

Reply:
I'm not sure this exemplar stuff is useful enough as specified. I'd like to support more than one exemplar. I'd like exemplars to carry a sample weight too.

Reply:
@jmacd I am going to create a separate PR, we can discuss there.